src/lib.rs

```diff
@@ -84,3 +126,97 @@ fn load_pem_certs(path: &Path) -> Result<Vec<CertificateDer<'static>>, Error> {
         })
         .collect()
 }
+
+/// Load certificate from certificate directory (what OpenSSL calls CAdir)
+///
+/// This directory can contain other files and directories. CAfile tends
```

Comment:
Ditto re: backticks for `CAfile`.

Reply:
I'm not sure. For me CAdir is more of a name used by OpenSSL. I usually use backticks only for some sort of code, either command line (`openssl reshash`), or a reference to the Rust code (`AStruct`, `func()`, `variable_name`).

Reply:
I don't feel strongly, leaving it as-is is OK with me :+1: